### PR TITLE
Feature/nl/position buttons

### PR DESCRIPTION
**Pull request by NicoleLatifi**
# Turing Index Pull Request Form

## Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Add buttons for different classrooms and position them accordingly.

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Testing (add-testing to project)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:

## Has This Been Tested?
- [ ] No
- [x] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:

Visually inspected.

# Message for reviewer:

Let's try some different styles for our ripple!

# Checklist:
- [ ] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/app/map/map.component.html [hunk 1]
```diff
@@ -4,7 +4,135 @@
 	<div class="map-parent">
 		<div class="button-container fe1">
 			<div class="button-box">
-				<a class='btn btn--white' style=" background-color: #caf0f8; font-size: 15px;" (click)="this.openDialog()">Nicoles ripple thingy</a>
+				<a class='btn btn--white' style=" background-color: #caf0f8; font-size: 15px;" (click)="this.openDialog()">Mod 1 FrontEnd</a>
+				<span style="--i:1;"></span>
```
Comment (JoshSevy) [inline src/app/map/map.component.html:8]: Great work @NicoleLatifi. Awesome addition.

**Review by JoshSevy — APPROVED** [review]
Great work! Home stretch here we come.

File: src/app/map/map.component.html [hunk 2]
```diff
@@ -4,7 +4,135 @@
 	<div class="map-parent">
 		<div class="button-container fe1">
 			<div class="button-box">
-				<a class='btn btn--white' style=" background-color: #caf0f8; font-size: 15px;" (click)="this.openDialog()">Nicoles ripple thingy</a>
+				<a class='btn btn--white' style=" background-color: #caf0f8; font-size: 15px;" (click)="this.openDialog()">Mod 1 FrontEnd</a>
```
Comment (Leasw144) [inline src/app/map/map.component.html:7]: Great job making the ripple effect, we'll need to work on making it conditional but not sure we'll get to it with the time left

**Review by Leasw144 — APPROVED** [review]
Great job!

**Pull request merged**